<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualControl</class>
 <widget class="QWidget" name="ManualControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>330</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Manual Control</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>368</width>
     <height>218</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonUp">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>Up</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/icons/Switch-Control-Up.png</normaloff>:/icons/Switch-Control-Up.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Up</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="buttonDown">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>Down</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/icons/Switch-Control-Down.png</normaloff>:/icons/Switch-Control-Down.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Down</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="buttonLeft">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>Left</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/icons/Switch-Control-Left.png</normaloff>:/icons/Switch-Control-Left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Left</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="buttonRight">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>Right</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/icons/Switch-Control-Right.png</normaloff>:/icons/Switch-Control-Right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Right</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="buttonCLK">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/icons/Switch-Control-Press.png</normaloff>:/icons/Switch-Control-Press.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonX">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>X</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/icons/Switch-Button-Top.png</normaloff>:/icons/Switch-Button-Top.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="shortcut">
         <string>S</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="buttonY">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>Y</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/icons/Switch-Button-Left.png</normaloff>:/icons/Switch-Button-Left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="shortcut">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="buttonA">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>A</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/icons/Switch-Button-Right.png</normaloff>:/icons/Switch-Button-Right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Z</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="buttonB">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>B</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/icons/Switch-Button-Bottom.png</normaloff>:/icons/Switch-Button-Bottom.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="shortcut">
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="3">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="buttonSelect">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>Select</string>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="buttonStart">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>Start</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="buttonCapture">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>Capture</string>
        </property>
        <property name="text">
         <string>Capture</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="buttonHome">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="statusTip">
         <string>Home</string>
        </property>
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>371</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="buttonZL">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ZL</string>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonZLZR">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ZL + ZR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonZR">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ZR</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonL">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>L</string>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonLR">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>L + R</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonR">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
